UnitTests/GitCommands.Tests/GitModuleTests.cs

```diff
@@ -679,18 +680,18 @@ public void GetStagedStatus(ObjectId firstRevision, ObjectId secondRevision, Obj
             Assert.AreEqual(status, stagedStatus);
         }
 
-        [Ignore("See https://github.com/gitextensions/gitextensions/issues/10387")]
         [Test]
         public void GetSubmodulesLocalPaths()
         {
-            List<CommonTestUtils.GitModuleTestHelper> moduleTestHelpers = new();
+            List<GitModuleTestHelper> moduleTestHelpers = new();
             try
             {
                 const int numModules = 4;
 
                 for (int i = 0; i < numModules; ++i)
                 {
-                    moduleTestHelpers.Add(new CommonTestUtils.GitModuleTestHelper($"repo{i}"));
+                    moduleTestHelpers.Add(new GitModuleTestHelper($"repo{i}"));
+                    Debug.WriteLine($"Repo[{i}]:{moduleTestHelpers[i].TemporaryPath}");
```

Comment:
Should this printout be kept?
(Setup as for [SubmoduleStatusProviderTests.cs](https://github.com/gitextensions/gitextensions/pull/10389/files#diff-e7dde1041edc7df2802b2896fac5e4004a158dfedc9334f60582133dfefa4c54) is a better fit.)

Reply:
Yes. It is there to help debugging tests in future. I did it to help track down issues when testing and capturing what folders are involved. I did debug to make sure it was only in debugging.